hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/TestMultiFS.java

```diff
@@ -102,28 +127,28 @@ public void readLocalWriteHDFS() throws Exception {
       // Write generated data to hdfs (only inserts)
       String readCommitTime = hdfsWriteClient.startCommit();
       LOG.info("Starting commit " + readCommitTime);
-      List<HoodieRecord> records = dataGen.generateInserts(readCommitTime, 100);
-      JavaRDD<HoodieRecord> writeRecords = jsc.parallelize(records, 1);
+      List<HoodieRecord> records = dataGen.generateInserts(readCommitTime, 10);
```

Comment:
some of the test assumes that records will span across all 3 partition paths (HoodieTestDataGenerator). if we reduce the num of records from 100 to 10, chances we might see some flakiness. something to keep in mind.

Reply:
this test case does not depend on all partitions having records. so i've only changed this one